_dprhtml/features/settings-dialog/init.js

```diff
@@ -1,10 +1,48 @@
 'use strict';
 
+const fontFamilyDefault = 'Verdana';
```

Comment:
Can we not read this from DPR_prefsD['colfont'], DPR_prefsD['colsize'] & DPR_prefsD['translits']?